lib/flatpak-game.lua

```diff
@@ -65,6 +65,7 @@ function identify(file)
 end
 
 function verify_gog(file)
+    print('Verifying game file.')
```

Comment:
We use 8-spaces tabs, along with spaces to align arguments. I don't think there are any cases where we only use spaces in our own code, but I'd take a patch that removed the use of those.